### PR TITLE
[Rust SDK] wasmedge-sdk v0.1.0

### PR DESCRIPTION
**Pull request by apepkuss**
In this PR `wasmedge-sdk` comes with `v0.1.0`.

### CODE REVIEW

**Comment by apepkuss** [comment]
@hydai Please help review this PR, thanks a lot!

**Comment by apepkuss** [comment]
@hydai 
One more thing, this PR should be merged after the core 0.10.0 and the sys 0.7.0 are officially released, as `wasmedge-sdk` in this PR has to refine the `wasmedge-sys` dep in its `Cargo.toml`.

**Review by hydai — CHANGES_REQUESTED** [review]

File: bindings/rust/wasmedge-sdk/src/vm.rs [hunk 1]
```diff
@@ -1,97 +1,312 @@
...
-///
-///     let results = vm.run("fib", &[5.into()])?;
-///
-///     assert_eq!(results.len(), 1);
-///     let result = results[0].as_i32().unwrap();
-///
-///     assert_eq!(result, 8);
-/// ```
-///
+// use crate::{
```
[the remark this answers is not in the document]
Comment (apepkuss) [reply bindings/rust/wasmedge-sdk/src/vm.rs:1]: Accepted. Thanks a lot!

File: bindings/rust/wasmedge-sdk/README.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
+# Overview
+
+The [wasmedge-sdk](https://crates.io/crates/wasmedge-sdk) crate defines a group of high-level Rust APIs, which are used to build up business applications.
```
[the remark this answers is not in the document]
Comment (apepkuss) [reply bindings/rust/wasmedge-sdk/README.md:3]: The motivation for adding this link is just to provide a one-click jump for the readers to direct them to the crate page on crates.io, where the readers can find the necessary information like how to import the crate into their projects and where the API reference is. Thanks a lot!

File: bindings/rust/wasmedge-sdk/src/config.rs [hunk 3]
```diff
@@ -1,28 +1,775 @@
...
+use wasmedge_sys as sys;
+use wasmedge_types::{CompilerOptimizationLevel, CompilerOutputFormat};
+
+/// Defines a builder for creating a [Config].
+#[derive(Debug)]
+pub struct ConfigBuilder {
+    common_config: CommonConfigOptions,
+    stat_config: Option<StatisticsConfigOptions>,
+    compiler_config: Option<CompilerConfigOptions>,
+    runtim_config: Option<RuntimeConfigOptions>,
```
Comment (hydai) [inline bindings/rust/wasmedge-sdk/src/config.rs:13]: `runtime`

File: bindings/rust/wasmedge-sdk/src/module.rs [hunk 4]
```diff
@@ -1,20 +1,218 @@
...
+        self.inner.count_of_imports()
+    }
+
+    /// Returns the [import types](crate::ImportType) of all imported WasmEdge instances in the [module](crate::Module).
+    pub fn imports(&self) -> Vec<ImportType> {
+        let mut imports = Vec::new();
+        for inner_import in self.inner.imports() {
+            let import = ImportType {
+                inner: inner_import,
+                // module: &self,
```
Comment (hydai) [inline bindings/rust/wasmedge-sdk/src/module.rs:89]: Redundant comment.

File: bindings/rust/wasmedge-sdk/src/module.rs [hunk 5]
```diff
@@ -1,20 +1,218 @@
...
+        self.inner.count_of_exports()
+    }
+
+    /// Returns the [export types](crate::ExportType) of all exported WasmEdge instances from the [module](crate::Module).
+    pub fn exports(&self) -> Vec<ExportType> {
+        let mut exports = Vec::new();
+        for inner_export in self.inner.exports() {
+            let export = ExportType {
+                inner: inner_export,
+                // module: &self,
```
Comment (hydai) [inline bindings/rust/wasmedge-sdk/src/module.rs:108]: Ditto.

**Review by hydai — CHANGES_REQUESTED** [review]

File: bindings/rust/wasmedge-sdk/src/config.rs (hunk 3 above)
Comment (apepkuss) [reply bindings/rust/wasmedge-sdk/src/config.rs:13]: Accepted. Thanks!

File: bindings/rust/wasmedge-sdk/src/module.rs (hunk 4 above)
Comment (apepkuss) [reply bindings/rust/wasmedge-sdk/src/module.rs:89]: Accepted. Thanks!

File: bindings/rust/wasmedge-sdk/src/module.rs (hunk 5 above)
Comment (apepkuss) [reply bindings/rust/wasmedge-sdk/src/module.rs:108]: Accepted. Thanks!

**Review by hydai — APPROVED** [review]

**Comment by apepkuss** [comment]
@hydai Thanks for the review!

**Pull request merged**